src/core/federation.ts

```diff
@@ -151,6 +151,65 @@ enum RTIMessageTypes {
      */
     MSG_TYPE_LOGICAL_TAG_COMPLETE = 9,
 
+    /////////// Messages used in lf_requestStop() ///////////////
```

Comment:
This is not the right place to put this. How about we create a wiki page and link to it?

Reply:
Done win https://github.com/lf-lang/reactor-ts/pull/111/commits/b399e4fa9de86f341d1313af173450b79b225ce1!

@lhstrh, How do you think about we add @ByeongGil-Jun as a member of LF repo so that he can edit the wiki next time?